apps/site/components/Common/Search/index.tsx

```diff
@@ -82,6 +82,9 @@ const SearchButton: FC = () => {
           HTMLTag: 'b',
           CSSClass: 'font-bold',
         }}
+        dictionary={Object.fromEntries(
```

Comment:
Is this TypeScript-safe? Like it verifies the keys are valid?

Reply:
Yes. Because I used `as const` when defining it, it's not a `string` type, but rather `"A" | "B" | "C"`, and for each one, it verifies it matches a valid parameter for the function.

Reply:
i.e what if I add `blah` to translationKeys should it complain about blah not being a valid orama key or not a key within our i18n?

Reply:
It'll complain about it not being a valid key.